mpa/cls/stage.py

```diff
@@ -297,17 +294,15 @@ def refine_tasks(train_cfg, meta, adapt_type):
     return model_tasks, old_tasks
 
 
-def refine_cls(train_cfg, meta, adapt_type):
+def refine_cls(train_cfg, data_classes, meta, adapt_type):
```

Comment:
I changed refine_cls and some codes to enable REPLACE mode in a right way. Before I change this, REPLACE mode did not work at all in incremental learning. I ask for your review :)